Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+RUN apt-get install -y wget sudo curl
+RUN wget https://github.com/EOSIO/eosio.cdt/releases/download/v1.5.0/eosio.cdt_1.5.0-1_amd64.deb
+RUN sudo apt install -y ./eosio.cdt_1.5.0-1_amd64.deb
+RUN wget https://github.com/EOSIO/eos/releases/download/v1.6.0/eosio_1.6.0-1-ubuntu-18.04_amd64.deb
+RUN sudo apt install -y ./eosio_1.6.0-1-ubuntu-18.04_amd64.deb
```

Comment:
<img width="800" alt="screen shot 2018-10-24 at 11 23 03 am" src="https://user-images.githubusercontent.com/1039525/51398041-908b8480-1b10-11e9-9d66-9b6998ec2fb5.png">

If you're using VSCode, of course.

Reply:
Thanks! Good to know this setting. I've updated the PR to add a new line